### DISCUSSION TITLE
WiFi Doorbell  firmware

### DISCUSSION THREAD

**Discussion by bagobones** [AT0myks/reolink-fw-archive#44] [Firmware sharing]
Beta/Dev firmware

[v3.0.0.2602_23081801 Reolink-Video-Doorbell-WiFi](https://support-d.reolink.com/attachments/token/fIg98yODUiUnAztUnQuvmRJtM/?name=DB_566128M5MP_W.2602_23081801.Reolink-Video-Doorbell-WiFi.OV05A10.5MP.WIFI8812.REOLINK.pak)

This is firmware I received via support that fixes the RTSP failing every 24hrs when used continuously. RTSP has been very stable with this version..

**Comment by univtex** [comment]
Getting a not found error with your link, would love to try this on my cam to fix this issue.

**Reply by bagobones** [reply]
Fixed the link https://support-d.reolink.com/attachments/token/fIg98yODUiUnAztUnQuvmRJtM/?name=DB_566128M5MP_W.2602_23081801.Reolink-Video-Doorbell-WiFi.OV05A10.5MP.WIFI8812.REOLINK.pak

**Comment by hapklaar** [comment]
Using this firmware as well, works much better than others. Even the latest firmware performs much worse on RTSP for me.

I asked Reolink support for a firmware with the new 3215 features and good RTSP support, now we wait....

**Comment by mx-vatte** [comment]
v3.0.0.3308_2405271823 is available via support request.
[https://drive.google.com/file/d/19LxC2tIbTUEBUNiwoFtAMPsC5MbF9YF5/view?usp=drive_link](https://drive.google.com/file/d/19LxC2tIbTUEBUNiwoFtAMPsC5MbF9YF5/view?usp=drive_link)
